### DISCUSSION TITLE
How to create a page sub component under pages without generating a route?

### DISCUSSION THREAD

**Discussion by tanhaoAli** [nuxt/nuxt#29682] [Questions (Nuxt 2)]
I want to create a page level sub component that is not defined in global components. How to create a sub component in pages like Vue? I have successfully created it, but it also generates a routing address. How can I avoid generating a routing address?

**Comment by tanhaoAli** [comment]
@victortolbert

**Comment by xjccc** [comment]
You could use [ignore](https://nuxt.com/docs/api/nuxt-config#ignore) have a try.

https://stackblitz.com/edit/nuxt-starter-rrb7uq?file=pages%2Findex.vue,app.vue,pages%2Ftest%2Findex.vue,pages%2Ftest%2Fcomp%2FTestComp.vue,nuxt.config.ts
